src/lib/utils.ts

```
@@ -1,6 +1,25 @@
+import { twMerge } from "tailwind-merge";
+import { AxiosError } from "axios";
+import { TRPCClientError } from "@trpc/client";
 
 export function cn(...inputs: ClassValue[]) {
-  return twMerge(clsx(inputs))
+  return twMerge(clsx(inputs));
+}
+
+export function getErrorMessage(
+  error: unknown,
+  defaultMessage: string = "Something went wrong. Please try again."
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)